podcasts/IAPHelper.swift

```diff
@@ -544,10 +556,21 @@ private extension IAPHelper {
                 offerType = "intro_offer"
             }
         }
+        if productId == .yearlyReferral {
+            offerType = "referral"
+        }
+        if let discount {
+            if discount.contains(".winback.") {
+                offerType = "winback"
+            } else if discount == "com.pocketcasts.plus.yearly.referral.promo" {
+                offerType = "referral"
+            }
+        }
```

Comment:
@SergioEstevao wdyt about using constants for this?

Reply:
Updated here: https://github.com/Automattic/pocket-casts-ios/pull/3152/commits/da67411d0c1a9f732f0348d806e7abf0ce15271f